spec.html

```diff
@@ -14380,7 +14380,7 @@
         1. Let _sl_ be the result of evaluating |StatementList|.
         1. ReturnIfAbrupt(_sl_).
         1. Let _s_ be the result of evaluating |StatementListItem|.
-        1. Return Completion(UpdateEmpty(_s_, _sl_)).
+        1. Return Completion(UpdateEmpty(_s_, _sl_.[[value]])).
```

Comment:
[ReturnIfAbrupt, step 2](http://tc39.github.io/ecma262/#sec-returnifabrupt) already takes care of unwrapping the completion value.